.github/workflows/slack.yml

```diff
@@ -0,0 +1,28 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Slack
+
+# Controls when the workflow will run
+on:
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    environment: DECIDE
```

Comment:
This should be changed to the name of the environment where the the secret is stored (in the last assignment Chiyi set up an environment named DECIDE where the secret was stored). If you named the environment DECIDE, then it is ok.

Reply:
Alright good point! I'll fix that